ggstat/agg.go

```diff
@@ -197,6 +197,15 @@ func AggSum(cols ...string) Aggregator {
 	return aggFn(vec.Sum, "sum ", cols...)
 }
 
+// AggPercentile returns an aggregate function that computes a
+// percentile of each of cols. The resulting columns will be named
+// "<prefix> <col>" and will have the same type as <col>.
+func AggPercentile(prefix string, percentile float64, cols ...string) Aggregator {
+	return aggFn(func(data []float64) float64 {
+		return stats.Sample{Xs: data}.Percentile(percentile)
+	}, prefix+" ", cols...)
```

Comment:
In the spirit of automatic sensible labeling, perhaps fmt.Sprintf("P%g ", percentile*100) for the prefix?